ckcp/openshift_dev_setup.sh

```diff
@@ -267,7 +270,7 @@ patches:
     sleep 5
```

Comment:
it does not originate from this PR but there are a couple of occurrences where $APP is used instead of $ns. Both variables contain the same value but ns was defined for the purpose and it is just an arbitrary decision to have the name of the namespace being the same as the application name: ckcp.